### PR TITLE
fix folder structure

### PR DESCRIPTION
**Pull request by Radhe2911**
fixed the folder structure

### CODE REVIEW

File: server/dist/controllers/userController.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        function step(result) { result.done ? resolve(result.value) : adopt(result.value).then(fulfilled, rejected); }
+        step((generator = generator.apply(thisArg, _arguments || [])).next());
+    });
+};
+const userDetails = (req, res, next) => __awaiter(void 0, void 0, void 0, function* () {
+    try {
+        const headers = req.headers.authorization;
+        console.log("headers", headers);
+        res.send(headers);
+        // const userDetails = await getUserDetails(userId, next)
```
Comment (tw-vineet) [inline server/dist/controllers/userController.js]: please remove commented code

File: server/dist/app.js
Comment (tw-vineet) [inline server/dist/app.js:1]: dist folder add in gitignore

File: server/src/app.ts [hunk 2]
```diff
@@ -18,7 +20,7 @@ app.use(fileUpload({
 }));
 
 //routes
-app.use("/api/auth", authRouter);
+app.use("/api", Routes)
 app.use("*", (req, res, next) => {
     next(new NotFoundError("Innvalid requested url"))
```
Comment (tw-vineet) [inline server/src/app.ts]: correct code smells 
`Invalid`

File: server/src/controllers/userController.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { Request, Response, NextFunction } from "express";
+import { getUserDetails } from "../models/Users.js";
+
+const userDetails = async (req: Request, res: Response, next: NextFunction) => {
+    try {
+
+
+        // const userDetails = await getUserDetails(userId, next)
+        // res.send(userDetails)
```
Comment (tw-vineet) [inline server/src/controllers/userController.ts]: remove commented code

File: server/src/middleware/errorHandler.ts [hunk 4]
```diff
@@ -1,9 +1,46 @@
...
+        super("Validation Error", false, 400);
+        this.errorData = data;
+        Object.setPrototypeOf(this, ValidationError.prototype)
+    }
+}
 
 // error handler middleware
 export const errorHandler: any = (err: Error, req: Request, res: Response, next: NextFunction) => {
-    console.log("error:", err)
+    console.log("error::", err)
```
Comment (tw-vineet) [inline server/src/middleware/errorHandler.ts:43]: remove console.log

File: server/dist/controllers/userController.js (hunk 1 above)
Comment (Radhe2911) [reply server/dist/controllers/userController.js]: Removed the commented code.

File: server/dist/app.js
Comment (Radhe2911) [reply server/dist/app.js:1]: Put the dist folder in .gitignore file

File: server/src/app.ts (hunk 2 above)
Comment (Radhe2911) [reply server/src/app.ts]: Spelling corrected.

File: server/src/middleware/errorHandler.ts (hunk 4 above)
Comment (Radhe2911) [reply server/src/middleware/errorHandler.ts:43]: This console is kept for bugging purpose for any kind of error. I will remove it at the the end.

**Pull request merged**